src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -180,6 +180,12 @@ You will often use these `astro` commands, or the scripts that run them, without
 
 Runs Astro's development server. This is a local HTTP server that doesn't bundle assets. It uses Hot Module Replacement (HMR) to update your browser as you save changes in your editor.
 
+You can use the following hotkeys in the terminal where the Astro develpment server is running.
+
+- `s + enter` to sync content layer.
```

Comment:
Might be nice to add a link to what `sync content layer` means. Do you have a suggestion?

Reply:
Great call! So I checked, and this is similar to our `astro sync` command (on the page) except it's specifically for reloading the types, and additionally the content, from the content layer. 

So, it could be as simple as:

> to sync the content layer data (content and types)

Reply:
Yeah that clarification makes sense to me!